data-otservbr-global/scripts/creaturescripts/quests/killing_in_the_name_of/boss_kill.lua

```diff
@@ -33,7 +33,7 @@ local taskBoss = {
 	[31] = "the horned fox",
 }
 
-local bossKillCount = Storage.Quest.U8_5.KillingInTheNameOf.BossKillCount.SnapperCount
+local bossKillCount = Storage.Quest.U8_5.KillingInTheNameOf.BossKillCount
```

Comment:
you are removing correctly storage 'SnapperCount'. BossKillCount it's an "array".